### ISSUE TITLE
File picker button is not focusable via tab key

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/shiny#2929]
### System details

Browser Version: any

Output of `sessionInfo()`:

``` r
library(shiny)
xfun::session_info()
#> R version 4.0.0 (2020-04-24)
#> Platform: x86_64-w64-mingw32/x64 (64-bit)
#> Running under: Windows 10 x64 (build 19041)
#> 
#> Locale:
#>   LC_COLLATE=English_United States.1252 
#>   LC_CTYPE=English_United States.1252   
#>   LC_MONETARY=English_United States.1252
#>   LC_NUMERIC=C                          
#>   LC_TIME=English_United States.1252    
#> system code page: 949
#> 
#> Package version:
#>   assertthat_0.2.1     backports_1.1.7      base64enc_0.1.3     
#>   BH_1.72.0.3          callr_3.4.3          cli_2.0.2           
#>   clipr_0.7.0          commonmark_1.7       compiler_4.0.0      
#>   crayon_1.3.4         digest_0.6.25        ellipsis_0.3.1      
#>   evaluate_0.14        fansi_0.4.1          fastmap_1.0.1       
#>   fs_1.4.1             glue_1.4.1           graphics_4.0.0      
#>   grDevices_4.0.0      highr_0.8            htmltools_0.5.0.9000
#>   httpuv_1.5.4         jsonlite_1.6.1       knitr_1.28.9        
#>   later_1.1.0.1        lifecycle_0.2.0      magrittr_1.5        
#>   markdown_1.1         methods_4.0.0        mime_0.9            
#>   pillar_1.4.4         pkgconfig_2.0.3      processx_3.4.2      
#>   promises_1.1.1       ps_1.3.3             purrr_0.3.4         
#>   R.cache_0.14.0       R.methodsS3_1.8.0    R.oo_1.23.0         
#>   R.utils_2.9.2        R6_2.4.1             Rcpp_1.0.4.6        
#>   rematch2_2.1.2       reprex_0.3.0.9001    rlang_0.4.6         
#>   rmarkdown_2.2.6      rprojroot_1.3.2      shiny_1.4.0.9003    
#>   sourcetools_0.1.7    stats_4.0.0          stringi_1.4.6       
#>   stringr_1.4.0        styler_1.3.2.9000    tibble_3.0.1        
#>   tinytex_0.23.2       tools_4.0.0          utf8_1.1.4          
#>   utils_4.0.0          vctrs_0.3.1          withr_2.2.0         
#>   xfun_0.14.4          xtable_1.8-4         yaml_2.2.1
```

<sup>Created on 2020-06-17 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0.9001)</sup>

### Example application *or* steps to reproduce the problem

1. Run the following code and press `Tab`/`Shift+Tab` key to focus file picker button.

``` r
library(shiny)

ui <- fluidPage(
  sidebarLayout(
    sidebarPanel(
      fileInput("file1", "Choose CSV File", accept = ".csv"),
      checkboxInput("header", "Header", TRUE)
    ),
    mainPanel(
      tableOutput("contents")
    )
  )
)

server <- function(input, output) {
  output$contents <- renderTable({
    file <- input$file1
    ext <- tools::file_ext(file$datapath)

    req(file)
    validate(need(ext == "csv", "Please upload a csv file"))

    read.csv(file$datapath, header = input$header)
  })
}

shinyApp(ui, server)
```

### Describe the problem in detail

In the example above, `Browse...` button does not have a `tabindex = "0"`, thus keyboard users cannot focus on the element via tab key.
